README.md

```diff
@@ -1 +1,68 @@
-# tf_label
\ No newline at end of file
+# tf_label
+
+Terraform module designed to generate consistent label names and tags for resources. Use `tf_label` to implement a strict naming convention. 
+
+All Cloud Posse modules use this module to ensure resources can be instantiated multiple times within an account.
+
+## Usage
```

Comment:
Add note regarding suggested order of input values in `id` output
`namespace` - `stage` - `name` - `attributes`